docs/developer/checkout.mdx

```diff
@@ -41,14 +41,17 @@ Depending on the chosen channel, the user will have access to different objects.
 If the checkout permission is assigned to the user, only this user and staff users with permission `MANAGE_CHECKOUTS` can query its data. Checkouts without assigned users can be queried modified using the token without additional permissions.
 
 ## Address Validation
+
 The checkout's mutations that accept an address as an input have a field that can turn off the address validation. This allows assigning a partial or not fully valid address to the checkout. Providing country code is mandatory for all addresses regardless of the rules provided in this input.
 
 The address [validation](developer/api-reference/inputs/checkout-address-validation-rules.mdx) input has two boolean fields:
+
 - `checkRequiredFields` - Determines if an error should be raised when the provided address doesn't have all the required fields. Set to `true` by default.
 - `checkFieldsFormat` - Determines if an error should be raised when the provided address doesn't match the expected format. Set to `true` by default.
 - `enableFieldsNormalization` - Determines if Saleor should apply normalization on address fields. Example: converting city field to uppercase letters. Set to `true` by default.
 
```

Comment:
```suggestion
- `checkRequiredFields` - raises an error when the provided address doesn't have all the required fields. Set to `true` by default.
- `checkFieldsFormat` - raises an error when the provided address doesn't match the expected format. Set to `true` by default.
- `enableFieldsNormalization` - Determines if Saleor should apply normalization on address fields. Example: converting a city field to uppercase letters. Set to `true` by default.

```

Reply:
@pawelpikus I'm not sure about this one so I left this untouched. Maybe @korycins has any comments

Reply:
Hhm, not sure of this. It is a flag to turn on/off raising exceptions by Saleor that are related to the addresses.
`checkRequiredFields - raises an error when the provided..` sounds to me like: "this flag will raise an exception" - but the flag is only a switch for Saleor 😅

Reply:
Yes, I see what you mean. Just wanted to get rid of the passive voice which is generally hard to read/follow. Perhaps:

```suggestion
- `checkRequiredFields` - signals Saleor to raise an error when the provided address doesn't have all the required fields. Set to `true` by default.
- `checkFieldsFormat` - signals Saleor to raise an error when the provided address doesn't match the expected format. Set to `true` by default.
- `enableFieldsNormalization` - Determines if Saleor should apply normalization on address fields. Example: converting city field to uppercase letters. Set to `true` by default.

```
What do you think?

Reply:
@SzymJ @korycins

Reply:
Sorry, I missed notification. It looks good!

Reply:
added @pawelpikus